sentinel/tests/unit/transitions/accept_patient_reports.js

```diff
@@ -256,7 +259,8 @@ describe('accept_patient_reports', () => {
     });
 
     it('does not associate visit to anything since it is not responding to a reminder', done => {
-      sinon.stub(transition, '_silenceReminders').callsArgWith(3, null, true);
+      sinon.stub(db.medic, 'post').callsArgWith(1, null, { _id: 'j' });
```

Comment:
Here as well, I think the stubs should actually simulate what the actual function will return.